packages/contracts/test/exchange/fill_order.ts

```diff
@@ -67,13 +67,7 @@ describe('FillOrder Tests', () => {
         const test = (fillScenarios: FillScenario[]) => {
             _.forEach(fillScenarios, fillScenario => {
                 const orderScenario = fillScenario.orderScenario;
-                const description = `Combinatorial OrderFill: ${orderScenario.feeRecipientScenario} ${
-                    orderScenario.makerAssetAmountScenario
-                } ${orderScenario.takerAssetAmountScenario} ${orderScenario.makerFeeScenario} ${
-                    orderScenario.takerFeeScenario
-                } ${orderScenario.expirationTimeSecondsScenario} ${orderScenario.makerAssetDataScenario} ${
-                    orderScenario.takerAssetDataScenario
-                }`;
+                const description = `Combinatorial OrderFill: ${JSON.stringify(fillScenario)}`;
```

Comment:
JSON.stringify(obj, null, 2)

Reply:
Since this is a test description, I purposefully left it more compact so that it doesn't take up tons of space. I'm willing to go back on this decision if you feel strongly.